.github/workflows/publish.yml

```
@@ -1,14 +1,16 @@
 name: Publish simulator archive
 
 on:
-  release:
-    types: [ published ]
+  push:
+    tags:
+      - '**'
```

Is this meant to be double?

```suggestion
      - '*'
```

---

Yes, otherwise tags with `/` in them will misteriously not run